src/__tests__/pdfUpload.integration.test.ts

```
@@ -67,4 +67,523 @@ describe('PDF upload & 27-section analysis (integration)', () => {
+
+    // Validate scenes structure
+    if (result.scenes) {
+      expect(["object", "string"]).toContain(typeof result.scenes);
+    }
+
+    // Validate that numeric fields are properly formatted
+    if (result.budget && typeof result.budget === "object") {
+      // Budget should contain numeric values if structured
+      Object.values(result.budget).forEach(value => {
+        if (typeof value === "string" && value.match(/^d+(.d+)?$/)) {
+          expect(parseFloat(value as string)).not.toBeNaN();
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)